ftva_lab_data/tests.py

```diff
@@ -750,3 +753,50 @@ def test_format_data_for_export(self):
             "Duplicated in source data, Incorrect inv no in filename",
             export_data[1]["status"],
         )
+
+
+class SetEmptyInvNoStatusTestCase(TestCase):
+    """Tests the set_empty_inv_no_status management command."""
+
+    fixtures = ["item_statuses.json"]
+
+    def test_set_empty_inv_no_status(self):
+        # Create a SheetImport object with an empty inventory number
+        SheetImport.objects.create(file_name="test_file", inventory_number="")
+
+        # Call the management command function directly
+        set_empty_inv_no_status()
+
+        # Check that the status was set correctly
+        updated_record = SheetImport.objects.get(file_name="test_file")
+        self.assertTrue(updated_record.status.filter(status="Invalid inv no").exists())
+
+    def test_set_empty_inv_no_status_existing_status(self):
+        # Create a SheetImport object with an empty inventory number
+        # and an existing 'Invalid inv no' status
```

Comment:
Should this comment be 'Needs review' status instead, to match the code at line 778?